From ab84cd1ea9bff4cb90771775cc0378df1825388c Mon Sep 17 00:00:00 2001
From: Grzegorz Jaszczyk <jaz@semihalf.com>
Date: Tue, 17 Apr 2018 15:29:51 +0200
Subject: [PATCH 0590/1051] arm64: dts: marvell: mcbin: add comphy references
 to PCIe ports

Adding phandles to the PCIe port will allow PCIe driver to configure
appropriate phys (Marvell comphys) accordingly.

Change-Id: Idbb5dfe2539498514fa79bb6de30a58ee710116e
Signed-off-by: Grzegorz Jaszczyk <jaz@semihalf.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts b/arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts
index 56fa44860909..1651ce386e6e 100644
--- a/arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts
+++ b/arch/arm64/boot/dts/marvell/armada-8040-mcbin.dts
@@ -184,6 +184,14 @@
 	num-lanes = <4>;
 	num-viewport = <8>;
 	reset-gpio = <&cp0_gpio1 20 GPIO_ACTIVE_LOW>;
+
+	/* Generic PHY, providing serdes lanes */
+	phys = <&cp0_comphy0 0
+		&cp0_comphy1 0
+		&cp0_comphy2 0
+		&cp0_comphy3 0>;
+	phy-names = "pcie-phy0", "pcie-phy1", "pcie-phy2", "pcie-phy3";
+
 	status = "okay";
 };
 
-- 
2.17.1

